autoload/commands/__load__

```diff
@@ -66,6 +66,7 @@ do
                     "$fg[green]$repo$reset_color"
                 ;;
         esac
+        exit $?
```

Comment:
Is `exit` required here? I doubt that, but if so I think it should be added to other commands as well, including `install`, `update` and `status`